Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -25,6 +25,9 @@ public Relationship(RelationshipTemplate relationshipTemplate, IdentityAddress a
         EnsureTargetIsNotSelf(relationshipTemplate, activeIdentity);
         EnsureNoActiveRelationshipToTargetExists(relationshipTemplate.CreatedBy, existingRelationships);
 
+        var listOfAllowedStatuses = new List<RelationshipStatus> { RelationshipStatus.Pending, RelationshipStatus.Active, RelationshipStatus.Rejected, RelationshipStatus.Revoked };
```

Comment:
is it just me, or is "allowedStatuses" a confusing/misleading name? 
1. All statuses are "allowed".
2. If you mean "allowed" in the sense of "existing relationships can have these statuses in order to be able to create a new relationship", then the list has to be inversed, doesn't it? Because if there is e.g. an `Active` relationship, we should not be allowed to create another one.

Reply:
Ok, I've changed the variable name to `listOfStatusesRelationshipCanBeInWhenCreatingNewRelationship`... I know it's long but at least it's unambiguous.

Changes made.

Reply:
That's not accurate. It only mentions one relationship. So it sounds as if you mean the relationship that is about to be created. 

Maybe we shouldn't create a variable, and just use it inline. So:


```c#
![RelationshipStatus.Pending, RelationshpStatus.Active, ...].Contains(r.Status)
```

Reply:
Ahh, I just saw that this isn't possible. :( I will think about it tomorrow.